### DISCUSSION TITLE
Frequency, invalid edit (Chirp)

### DISCUSSION THREAD

**Discussion by pietss** [egzumer/uv-k5-firmware-custom#213] [General]
I apologize for the question which is perhaps trivial but I didn't understand what the problem is, which probably comes from Chirp.
It seems that using Chirp it is not possible to memorize the frequencies ranging, approximately (the range could be greater), from 77.00 to 79.00
In chirp I checked the box: Setting: Driver information: Limit disabled for modified firmware; but the window appears saying: Invalid edit: frequency [….] is of out of supported range….
Can this be resolved somehow? 
Many thanks

**Comment by Arikshten** [comment]
It looks like there is already work being done on adding an appropriate memory map to Chirp 

https://github.com/egzumer/uv-k5-firmware-custom/discussions/183

This might resolve any issues.

**Comment by egzumer** [comment]
This is a chirp issue, it doesn't support 76 - 108. Has nothing to do with any modifications in this firmware. The frequency table for unlocked frequencies is incorrect in chirp.
New driver specifically for this version of firmware is on its way. It should be fixed there.

**Comment by pietss** [comment]
Fine thanks a lot.
